config/hpa-autoscaling/controller.yaml

```diff
@@ -91,9 +94,11 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    app: autoscaler-hpa
```

Comment:
Can we leave this app label to be consistent?

You're right about this not affecting the selector - I'm just unsure if users use this label in their CI/CD system and we should just give notice we'll be removing it release after next.

Reply:
ok, added it back in